api/godog/retry/retry_test.go

```diff
@@ -0,0 +1,63 @@
+package retry
```

Comment:
I think it is nice to use retry_test when writing tests. this way you will experience how the package is perceived from the consumer.